src/Razor/src/Microsoft.VisualStudio.LanguageServerClient.Razor/HtmlCSharp/FindAllReferencesHandler.cs

```diff
@@ -208,7 +208,10 @@ async Task DelayAfterLastNotifyAsync(CancellationToken cancellationToken)
             progress.Report(remappedResults);
         }
 
-        private async Task<VSInternalReferenceItem[]> RemapReferenceItemsAsync(VSInternalReferenceItem[] result, CancellationToken cancellationToken)
+        private Task<VSInternalReferenceItem[]> RemapReferenceItemsAsync(VSInternalReferenceItem[] result, CancellationToken cancellationToken)
+            => RemapReferenceItemsAsync(result, _documentMappingProvider, _documentManager, cancellationToken);
+
+        public static async Task<VSInternalReferenceItem[]> RemapReferenceItemsAsync(VSInternalReferenceItem[] result, LSPDocumentMappingProvider documentMappingProvider, LSPDocumentManager documentManager, CancellationToken cancellationToken)
```

Comment:
The only place I see using `RemapReferenceItemsAsync` is using the non-static version of this, so why did we need a static version?

Reply:
I'm calling the static version when handling VSReferenceItems in the FindImplementations handlee